tests/tests/conftest.py

```diff
@@ -1,9 +1,29 @@
 import os
 import pytest
+import subprocess
 
 import Qemu
 import util
 
+script_path=os.path.dirname(os.path.realpath(__file__))
+
+# Is platform registered for quote generation
+def is_platform_registered():
+    try:
+        subprocess.check_call([f'{script_path}/../../attestation/check-registration.sh'])
+    except:
+        return 0
```

Comment:
Can you use `False` and `True` here? This confused me at first because I saw it was a shell script being called and therefore assumed zero should mean success and non-zero should mean failure.

Reply:
Done : 9eee5b3dd53d405d6c8923f37771668827b6d5b6